src/Android/Avalonia.Android/Platform/SkiaPlatform/TopLevelImpl.cs

```diff
@@ -55,6 +57,9 @@ public TopLevelImpl(AvaloniaView avaloniaView, bool placeOnTop = false)
             _gl = new EglGlPlatformSurface(this);
             _framebuffer = new FramebufferManager(this);
 
+            (AvaloniaLocator.Current.GetRequiredService<IClipboard>() as ClipboardImpl)?
```

Comment:
We can have multiple toplevels. Clipboard should be removed from the "global variables" scope to be TopLevel-scoped, like the rest of platform services.